runtime/common/src/changes.rs

```diff
@@ -0,0 +1,104 @@
+use cfg_primitives::SECONDS_PER_WEEK;
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::RuntimeDebug;
+use pallet_loans::entities::changes::{Change as LoansChange, InternalMutation, LoanMutation};
+use pallet_pool_system::pool_types::changes::{PoolChangeProposal, Requirement};
+use scale_info::TypeInfo;
+use sp_runtime::DispatchError;
+use sp_std::vec;
+
+/// Auxiliar type to carry all pallets bounds used by RuntimeChange
+pub trait Changeable: pallet_loans::Config {}
+impl<T: pallet_loans::Config> Changeable for T {}
+
+/// A change done in the runtime, shared between pallets
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub enum RuntimeChange<T: Changeable, Options: Clone = ()> {
```

Comment:
This is the key. If none option is passed, `RuntimeChange<T>`, the behavior is the expected one. If `FastDelay` option is passed, `RuntimeChange<T, FastDelay>` the behavior is hacked to be faster.